pcm_fitModelCrossval.m

```diff
@@ -152,20 +152,20 @@
     end;
     
     % Get starting guess for theta if not provided
-    if (isfield(M(m),'theta0'))
+    if (isfield(M(m),'theta0')) && ~(isempty(M(m).theta0))
```

Comment:
Added the "is not empty" clause to this line to avoid crashes if Model structure contains empty theta0 field for a model. Thought this could occur in cases where someone defines numerous types of models and accidentally creates an empty field.